spring-data-microservice-transaction-sample/README.md

```diff
@@ -0,0 +1,550 @@
+# Microservice Transaction Sample
+
+This tutorial describes how to create a sample Spring Boot application for microservices by using Spring Data JDBC for ScalarDB with Two-phase Commit Transactions of ScalarDB.
+
+For details, please see [Two-phase Commit Transactions in ScalarDB](https://github.com/scalar-labs/scalardb/tree/master/docs/two-phase-commit-transactions.md) and [Guide of Spring Data JDBC for ScalarDB](https://github.com/scalar-labs/scalardb-sql/blob/main/docs/spring-data-guide.md).
+
+## Prerequisites
+
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+In addition, you need access to the [ScalarDB SQL GitHub repository](https://github.com/scalar-labs/scalardb-sql) and [Packages in ScalarDB SQL repository](https://github.com/orgs/scalar-labs/packages?repo_name=scalardb-sql).
+These repositories are available only to users with a commercial license and permission.
+To get a license and permission, please [contact us](https://scalar-labs.com/contact_us/).
+
+You also need the `gpr.user` property for your GitHub username and the `gpr.key` property for your personal access token.
+You must either add these properties in `~/.gradle/gradle.properties` or specify the properties by using the `-P` option when running the `./gradlew` command as follows:
+
+```shell
+$ ./gradlew run ... -Pgpr.user=<YOUR_GITHUB_USERNAME> -Pgpr.key=<YOUR_PERSONAL_ACCESS_TOKEN>
+````
+
+Or you can also use environment variables, `USERNAME` for your GitHub username and `TOKEN` for your personal access token.
+
+```shell
+$ export USERNAME=<YOUR_GITHUB_USERNAME>
+$ export TOKEN=<YOUR_PERSONAL_ACCESS_TOKEN>
+```
+
+For more details, please see [Install - ScalarDB SQL](https://github.com/scalar-labs/scalardb-sql#install).
+
+## Sample application
+
+### Overview
+
+This tutorial describes how to create a Spring Boot sample application for Microservice Transaction for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Two-phase Commit Transactions in ScalarDB.
+
+There are two microservices called *Customer Service* and *Order Service* based on the [*Database-per-service* pattern](https://microservices.io/patterns/data/database-per-service.html) in this sample application.
```

Comment:
```suggestion
There are two microservices called a *Customer Service* and an *Order Service* based on the [*Database-per-service* pattern](https://microservices.io/patterns/data/database-per-service.html) in this sample application.
```

Probably.
@josh-wong Is it correct?

Reply:
"a Customer Service and an Order Service" sounds a bit to me that there are multiple Customer Services and Order Services. What do you think?

Reply:
In this sentence, I would use "a *Customer Service* and an *Order Service*". After we introduce them like this and use "the Customer Service" and "the Order Service" in subsequent sentences, we are establishing that there is one service for each.

Reply:
Thanks! I'll update the sentence.

Reply:
Done! https://github.com/scalar-labs/scalardb-samples/pull/37/commits/1c763bc10902d094a05ef3d0e7ffee7329c7ad84